tests/gd2/php_bug_72339.c

```diff
@@ -8,14 +8,26 @@ int main()
 {
 	gdImagePtr im;
 	FILE *fp;
-	
-	fp = gdTestFileOpen2("gd2", "php_bug_72339_exp.gd2");
-	im = gdImageCreateFromGd2(fp);
-	if (im == NULL) {
-		return 0;
-	} else {
-		gdTestErrorMsg("Image should have failed to be loaded\n");
-		return 1;
+	long i;
+
+	/* we're creating the test image dynamically, due to it's size */
+	fp = gdTestTempFp();
+	fputc(0x00, fp); fputc(0x02, fp);
+	fputc(0x00, fp); fputc(0x01, fp);
+	fputc(0x00, fp); fputc(0x01, fp);
+	fputc(0x00, fp); fputc(0x40, fp);
+	fputc(0x00, fp); fputc(0x02, fp);
+	fputc(0x54, fp); fputc(0xA0, fp);
+	fputc(0x5B, fp); fputc(0x00, fp);
+	for (i = 0; i < 0x4000000L; i++) {
+		fputc(0x41, fp);
 	}
+	rewind(fp);
+
+	im = gdImageCreateFromGd2(fp);
+	fclose(fp);
+	
+	gdTestAssertMsg(im == NULL, "Image should have failed to be loaded%c", '\n');
```

Comment:
That's a workaround, because gdTestAssertMsg() fails to compile with less than 3 arguments.